src/task/MonitorTask.ts

```diff
@@ -345,7 +363,7 @@ export default class MonitorTask
       event.type === SignalingClientEventType.WebSocketFailed
     ) {
       if (!this.hasSignalingError) {
-        this.context.eventController?.pushMeetingState('signalingDropped');
+        this.context.eventController?.publishEvent('signalingDropped');
```

Comment:
Sorry, I should've mentioned in the previous revision. You can also include attributes for the `signalingDropped` event. You may create a private helper method to generate the same attributes for `signalingDropped` and `receivingAudioDropped`.

Reply:
Updated Signalling dropped event to contain audioVieo Attributes